llvm/include/llvm/Analysis/IR2Vec.h

```diff
@@ -306,6 +309,20 @@ class LLVM_ABI SymbolicEmbedder : public Embedder {
   }
 };
 
+/// Class for computing the Flow-aware embeddings of IR2Vec.
+/// Flow-aware embeddings build on the vocabulary, just like Symbolic
+/// embeddings, and additionally capture the flow information in the IR.
+class LLVM_ABI FlowAwareEmbedder : public Embedder {
+private:
+  void computeEmbeddings(const BasicBlock &BB) const override;
+
+public:
+  FlowAwareEmbedder(const Function &F, const Vocabulary &Vocab)
+      : Embedder(F, Vocab) {
+    FuncVector = Embedding(Dimension, 0);
```

Comment:
initialize in the initializer list? `FuncVector(Dimension, 0)`

same in the Symbolic, now that I see it

another option would be to let the base class do it, seems both cases (flow and symbolic) would do the same thing?